### ISSUE TITLE
Location Field doesn't work anymore

### ISSUE THREAD

**Issue by Maxican** [flopperj/osm-map-elementor#28]
Clicking into the field locks it. A circled exclamation mark appears. The page I noticed this uses the new Flex Container feature. 
On older pages with sections and columns it still works. 
Current workaround would be to set a pin in google maps and then copy & paste its coordinates.

WordPress 6.1.3
Elementor 3.13.4
Elementor Pro 3.13.2

**Comment by flopperj** [comment]
@Maxican Thank you for your feedback. Quick question, do you have an up-to date Google Maps API key? I just did a quick test in my local env and all seems to be working as expected.

Would you also please share a screenshot or screen capture video of any console errors you're seeing that might help in debugging the issue,

Thanks,

**Comment by Maxican** [comment]
@flopperj Thank you for taking the time and pointing me in the right direction. You're absolutely right. Everything's okay. I checked the console and there it was: RefererNotAllowedMapError. I just had to remove the dev. subdomain. I totally forgot about that restriction in my API settings and the fact that I'm able to edit existing locations although the website was moved to the live domain in the meantime was extra confusing.
So, thank again! 👊🏽
